### DISCUSSION TITLE
连接/找 不到 管理界面

### DISCUSSION THREAD

**Discussion by Jerry-2022** [istoreos/istoreos#1794] [Q&A]
各位好哈，请教一下：
硬件：CMCC RAX3000M
系统：iStoreOS 25.03

了一条CMCC宽带，所以想做个双网。

RAX3000M连接两个光猫的2.5G口，打算让光猫拨号，RAX3000M直接连到光猫的2.5LAN口。

在iStoreOS管理页面>网络>接口那里，把eth1和eth0都改成了lan，就出问题了。

现在是接上光猫，输入192.168.1.1，就进入了光猫的网关，再也找不到iStoreOS的管理页面了...

各位有什么经验和建议么？？？

多谢

**Comment by w9315273** [comment]
`了一条CMCC宽带，所以想做个双网。`
暂且认为是字没复制全 <br>

`RAX3000M连接两个光猫的2.5G口，打算让光猫拨号，RAX3000M直接连到光猫的2.5LAN口。`
你这个描述我读了几十遍 楞是没捋清 【`两个光猫` `的2.5G口`】还是 【`两个` `光猫的2.5G口`】
结合上一句来看的话是 `两个光猫` 想要用路由器做 `“双网叠加”` ? 是这样理解的吧? <br>

`在iStoreOS管理页面>网络>接口那里，把eth1和eth0都改成了lan，就出问题了。`
这个拓扑下你是做不了聚合的 负载均衡也不建议做 只建议搞搞分流 走各自擅长的线路 <br>

你这个要恢复的话要跟你讲一大堆 我建议你直接重装

**Comment by jjm2473** [comment]
RAX3000M是啥机型啊，而且我们也没发布25.03或者23.05固件啊，只有x86有测试版的23.05固件。

把两个口都桥接到lan的话，就只有一个dhcp有效了，如果你要访问不同网段的设备，只能手动在电脑上添加静态IP了。

**Reply by shaokn** [reply]
咨询一下，“只有x86有测试版的23.05固件。” ， 这个固件在哪里下载啊？

**Reply by jjm2473** [reply]
现在都24.10了，别再关注23.05

**Reply by shaokn** [reply]
写错了，写错了，是25.03有吗？ 主要是有个duperemove的可以给btrfs去重的软件 https://github.com/ktgeek/packages/blob/master/utils/duperemove/Makefile

**Reply by jjm2473** [reply]
真不知道你在说什么，哪有25.03？

而且 https://github.com/ktgeek/packages/blob/master/utils/duperemove/Makefile 也不是openwrt官方的软件啊，就算openwrt有25.03版本，也不会有这个软件的。

**Reply by shaokn** [reply]
> 真不知道你在说什么，哪有25.03？
> 
> 而且 https://github.com/ktgeek/packages/blob/master/utils/duperemove/Makefile 也不是openwrt官方的软件啊，就算openwrt有25.03版本，也不会有这个软件的。

搞错了搞错了，我以为25.03是openwrt的版本，原来是那个第三方的X-wrt。就是那个把opkg命令换成了apk的，啥时候istoreOS也会更换吗？ 我编译了这个duperemove.apk可以安装的，ipk就失败了。这个是给btrfs分区做dedup的，希望可以加到istoreOS里面。当然目前也能用需要每天定时到那个ubuntu2的docker里面跑duperemove -rd /mnt/ssd，希望istoreOS作为一个轻nas可以支持btrfs的dedup。

**Reply by jjm2473** [reply]
那你用24.10以下的sdk编译下，就有ipk了。apk现在根本都还没正式使用，你觉得我们会切换到apk吗？这不是自找麻烦？还是说你有办法同时兼容apk和ipk？

**Reply by shaokn** [reply]
> 那你用24.10以下的sdk编译下，就有ipk了。apk现在根本都还没正式使用，你觉得我们会切换到apk吗？这不是自找麻烦？还是说你有办法同时兼容apk和ipk？

编译不成功啊…… 后来我看到作者是25年3月更新的代码，就试了一下apk就可以了。 然后，我的意思是，可以把duperemove加到istoreOS里面吗？作为一个轻nas，需要这个功能（btrfs dedup）

**Reply by jjm2473** [reply]
一般情况下，编译文件夹有可执行文件，拷贝这些可执行文件到你的系统里，然后安装依赖就能跑起来了。

还有，就算使用x-wrt编译，也是可以选择opkg做包管理器的。

此外，这个软件这么简单，24.10下编译失败不会尝试解决编译问题吗？

**Reply by jjm2473** [reply]
你测试过这个软件可用吗，我看作者在 openwrt 发了个 pr 又自己关掉了 https://github.com/openwrt/packages/pull/26167

**Reply by jjm2473** [reply]
我在24.10编译没遇到任何问题，不知道你是怎么编译的。这是我编译出来的包 
[duperemove_0.15.1-r1_x86_64.ipk.zip](https://github.com/user-attachments/files/20876292/duperemove_0.15.1-r1_x86_64.ipk.zip)

**Reply by shaokn** [reply]
> 我在24.10编译没遇到任何问题，不知道你是怎么编译的。这是我编译出来的包 [duperemove_0.15.1-r1_x86_64.ipk.zip](https://github.com/user-attachments/files/20876292/duperemove_0.15.1-r1_x86_64.ipk.zip)

感谢🙏 用了一下可以的，放到计划任务里面每天凌晨运行了。 希望官方可以早日加进来。
![image](https://github.com/user-attachments/assets/aeecee32-6e1c-4498-bd3a-9a4301f842a0)
